src/WebUI/Pods/PodsLeftPanel.tsx

```diff
@@ -54,44 +52,14 @@ export class PodsLeftPanel extends BaseComponent<IPodsLeftPanelProperties> {
         }
     }
 
-    private _getPanelHeaderContent(): JSX.Element {
-        const metadata: V1ObjectMeta = this.props.parentMetaData;
-        const columns: ITableColumn<any>[] = [
-            {
-                id: "PodsHeaderTable",
-                name: metadata.name,
-                width: -100,
-                className: "s-key",
-                minWidth: 180,
-                renderCell: PodsLeftPanel._renderValueCell
-            }
-        ];
-        const tableItems = new ArrayItemProvider<any>([
-            { key: Resources.KindText, value: this.props.parentKind },
-            { key: Resources.Created, value: metadata.creationTimestamp ? new Date(metadata.creationTimestamp) : new Date().getTime() },
-            { key: Resources.LabelsText, value: metadata.labels || {} },
-            { key: Resources.ImageText, value: Utils.getPodImageName(this.props.podTemplate) }
-        ]);
-
-        return (
-            <Card className="pods-left-pane-header-table" titleProps={{ text: Resources.SummaryText }}>
-                <Table
-                    className="s-full-details"
-                    id={format("s-full-details-{0}", metadata.uid)}
-                    showHeader={false}
-                    showLines={false}
-                    singleClickActivation={false}
-                    itemProvider={tableItems}
-                    pageSize={tableItems.length}
-                    columns={columns}
-                />
-            </Card>
-        );
+    private _getHeader(): JSX.Element | null {
+        /*Add back button here when we have support for the same*/
+        return (<h2 className="pod-left-panel-header">{this.props.parentName}</h2>);
```

Comment:
is h2 the right way to use here?

Reply:
For all headings across different pages we are using h2, so I am using same here for consistency